libminifi/test/unit/WindowsCertStoreLocationTests.cpp

```diff
@@ -51,4 +51,10 @@ TEST_CASE("CurrentUser and LocalMachine are among the allowed values of WindowsC
   CHECK_FALSE(ranges::contains(allowed_locations, "SomebodyElsesComputer"));
 }
 
+#else
+#include "../Catch.h"
+TEST_CASE("WindowsCertStoreLocation") {
```

Comment:
This could be removed as well.

Reply:
I removed this in https://github.com/apache/nifi-minifi-cpp/pull/1615/commits/44c7d79b3d130f483271f92ef7b8b224533e4137